sys/parsoid.js

```diff
@@ -166,6 +166,16 @@ function compileReRenderBlacklist(blacklist) {
     return result;
 }
 
+function stripAcceptFromVary(res) {
+    if (res.headers.vary) {
```

Comment:
Let's make this `if (res && res.headers && res.headers.vary)` to be on the safe side.

Reply:
Done.